src/Messages.js

```diff
@@ -142,7 +142,13 @@ export default defineMessages({
   ruleIsDisabledBody: {
     id: 'ruleIsDisabledBody',
     description: 'Explaining that the recommendationis disabled',
-    defaultMessage: 'This recommendation has been disabled and has no results.',
+    defaultMessage: 'This recommendation was disabled and has no results.',
+  },
+  ruleIsDisabledBodyWithJustification: {
+    id: 'ruleIsDisabledBodyWithJustification',
+    description: 'Explaining that the recommendationis disabled',
```

Comment:
Minor: there is a spelling error on line: 144

Reply:
Nice catch! Fixed that!